src/geff/writer_helper.py

```diff
@@ -0,0 +1,104 @@
+import abc
+from typing import Any, Sequence
+
+import numpy as np
+import zarr
+from numpy.typing import ArrayLike
+
+
+class BaseWriterHelper(abc.ABC):
+    """
+    Base class for writing geff attributes to a zarr group.
+
+    Important:
+        This function does not create the `ids` attribute.
+
+    Args:
+        group: The zarr group to write the attribute to.
```

Comment:
Just checking my understanding and then maybe we need to add more documentation. This is assumed to be an already opened zarr group which is either nodes or edges inside of the root geff group?